servicelayer/worker.py

```diff
@@ -143,14 +153,14 @@ def retry(self, task):
             log.warning(
                 f"Queueing failed task for retry #{retry_count}/{settings.WORKER_RETRY}..."  # noqa
             )
-            TASK_FAILED.labels(task.stage.stage, retries, False).inc()
+            TASKS_FAILED.labels(task.stage.stage, retries, False).inc()
             task.context["retries"] = retry_count
             task.stage.queue(task.payload, task.context)
         else:
             log.warning(
                 f"Failed task, exhausted retry count of {settings.WORKER_RETRY}"
             )
-            TASK_FAILED.labels(task.stage.stage, retries, True).inc()
+            TASKS_FAILED.labels(task.stage.stage, retries, True).inc()
```

Comment:
For the sake of maintainabilty looking forward, could we use keyword arguments to make this a bit more readable? ([The docs](https://prometheus.github.io/client_python/instrumenting/labels/) suggest this works)

```suggestion
            TASKS_FAILED.labels(stage=task.stage.stage, retries=retries, failed_permanently=True).inc()
```

Reply:
Oh yes, absolutely. I’ve been wondering about that before, not sure how I’ve missed that… 🙃 I’ll make the change in servicelayer and also in ingest-file and Aleph.

Reply:
@stchris Done! :)

Reply:
Thanks, Till!